custom-metrics-stackdriver-adapter/pkg/adapter/coreprovider/provider.go

```diff
@@ -49,9 +50,83 @@ type coreClientInterface interface {
 	getNodeRAM(nodesNames []string) (map[string]resource.Quantity, map[string]api.TimeInfo, error)
 }
 
+func (p *CoreProvider) GetPodMetrics(pods ...*metav1.PartialObjectMetadata) ([]metrics.PodMetrics, error) {
```

Comment:
NIT: comment exported methods for readability and go linters. same thing for GetNodeMetrics

Reply:
Done.